### ISSUE TITLE
Add Predictor class to predict games via ML purely via Numpy

### ISSUE THREAD

**Issue by kayoMichael** [kayoMichael/premier_league#14]
since the package has access to extensive advanced metrics of soccer matches. Predictions by Neural network can be trained and saved as a class. 
To keep the package as small as possible, it should be implemented purely in numpy. 

Since there are under 20,000 rows of advance metrics in the dataset, any deep learning models other than a basic pass through neural net (1~3 hidden layers) will lead to over-fitting. So libraries like Pytorch or Tensorflow is overkill since any advanced forms of neural net will not be necessary.

Things that need to be manually implemented compared to external libraries (That I can think of):
- backpropagation
- drop out
- loss
- learning rate initialization
